lib/ansible/modules/cloud/cloudscale/cloudscale_floating_ip.py

```diff
@@ -137,6 +152,11 @@
   returned: success
   type: str
   sample: 185.98.122.176
+region:
+  description: The region of the floating IP address or network.
```

Comment:
Maybe just call it floating IP, since that's what it's called in the API documentation.